pkg/repo/migrations/v3_4.go

```diff
@@ -15,45 +19,84 @@ import (
 // - if an installationID is present in the config its value is persisted to system_metadata.yaml.
 // - removes update.json if the file is present.
 // - removes repo.version.
-var V3Migration = repo.NewMigration(
+// - creates a new directory .bacalhau/compute_store/executions which contains the content of ./bacalhau/execution_store
+// - removes ./bacalhau/execution_store
+// - iff a user has configured a user key path, the configured value is copied to .bacalhau/user_id.pem
+// - iff a user has configured a auth tokens path, the configured value is copied to .bacalhau/tokens.json
+var V3Migration = StagedMigration(
```

Comment:
Given we haven't released since this migration was added I chose to update it to accommodate this change